src/contrib/cluster/Akka.Cluster.Tools/PublishSubscribe/DistributedPubSubMediator.cs

```diff
@@ -545,7 +659,7 @@ IEnumerable<IActorRef> Refs()
             }
         }
 
-        private void PublishToEachGroup(string path, Publish publish)
+        private void PublishToEachGroup(string path, IWrappedMessage publish)
```

Comment:
IMHO, do a common interface for both of the `Publish` message types and handle that instead.

Reply:
This can't be done because `PublishMessage` is also used to send `SendAll`